File: 백준/Bronze/2566. 최댓값/최댓값.py

arr = []
for _ in range(9):
    temp = list(map(int, input().split()))
    arr.append(temp)
max = 0
x, y = 0, 0
for i in range(9):
    for j in range(9):
        if max < arr[i][j]:
            max = arr[i][j]
            x = j
            y = i
            

print(max)
print(y+1, x+1)

File: 백준/Bronze/2566. 최댓값/최댓값.cc
#include <iostream>

using namespace std;
int main()
{
    int arr[9][9];
    int max = 0;
    int x, y = 0;
    for(int i = 0; i < 9; i++){
        for(int j= 0; j < 9; j++){
            cin >> arr[i][j];
            if(arr[i][j] > max){
                max = arr[i][j];
                x = i;
                y = j;
            }
        }
    }

    cout << max << endl << x+1 << ' ' << y+1;
    
}